app/views/content/shared/quotes/_quote_history_jack.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="quote-jack">
+ <%= render Content::QuoteComponent.new(
+      text: "I enjoy how as a teacher you can help pass on your passion for a subject onto students and inspire them to want to continue to study the subject whilst also helping them develop their own sense of curiosity.",
```

Comment:
Might need a couple of commas in here!

Reply:
I agree - and also think you might want to change `pass on your passion for a subject onto students` to `pass on your passion for a subject to students`

ie change `onto` to `to` as the sentence already contains `on`?